extensions/teams/package.json

```diff
@@ -27,12 +27,12 @@
         "restify": "^11.1.0"
     },
     "devDependencies": {
+        "@types/node": "^18.19.26",
         "@types/restify": "^8.5.5",
-        "@types/node": "^14.0.0",
         "env-cmd": "^10.1.0",
-        "ts-node": "^10.9.2",
-        "typescript": "^5.4.3",
         "nodemon": "^3.1.0",
-        "shx": "^0.3.3"
+        "shx": "^0.3.3",
+        "ts-node": "^10.9.2",
+        "typescript": "^5.4.3"
     }
-}
```

Comment:
I can never work this out in the editor - but can we have a blank line at the end please? Feel free to resolve if already there! :-)

Reply:
The PR is adding it 😄